### PR TITLE
Tweak/etp 702 hide edit column on print

### PR DESCRIPTION
**Pull request by redscar**
### 🎫 Ticket

[ETP-702] 

### 🗒️ Description

Removed the Edit column from displaying when printing and updated the column count for resizing the table when printing.

Added class to the printer styles for hiding columns

### 🎥 Artifacts 

image

https://user-images.githubusercontent.com/12241059/126359695-54f6cb3c-409d-4050-939f-606148cf9fdb.mp4

### ✔️ Checklist
- [x] I've included a changelog entry both in readme.txt and changelog.txt files. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ETP-702]: https://theeventscalendar.atlassian.net/browse/ETP-702

### CODE REVIEW

**Review by bordoni — DISMISSED** [review]

File: readme.txt [hunk 1]
```diff
@@ -182,7 +182,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 = [5.1.6] TBD =
 
-
+* Tweak - Removed the edit column when printing the Attendees list. [ETP-ETP-702]
```
Comment (juanfra) [inline readme.txt:185]:
```suggestion
* Tweak - Removed the edit column when printing the Attendees list. [ETP-702]
```
The ID of the ticket has duplicated contents.

**Review by juanfra — CHANGES_REQUESTED** [review]
Can we have a screencast as artifacts? Switching the columns we have on that view, by enabling/disabling them on the "Screen options" section? (top right part of the screen). I think there's also a merge conflict on the readme.

File: readme.txt [hunk 2]
```diff
@@ -180,6 +180,9 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 = [5.2.1] TBD =
 
+* Tweak - Removed the edit column when printing the Attendees list. [ETP-702]
+
+=======
```
Comment (juanfra) [inline readme.txt:185]: We need to remove these two lines.

File: readme.txt [hunk 3]
```diff
@@ -192,6 +195,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 * Fix - Prevent Tribe Commerce "Confirmation email sender name" from displaying improperly when a single quote is added. [ET-1134]
 * Language - 115 new strings added, 118 updated, 0 fuzzied, and 0 obsoleted
 
+
```
Comment (juanfra) [inline readme.txt:198]: We shouldn't add this new blank line here.

**Review by juanfra — APPROVED** [review]

File: readme.txt (hunk 2 above)
Comment (bordoni) [inline readme.txt:185]: We still need to remove this `======`

**Review by bordoni — CHANGES_REQUESTED** [review]

**Review by bordoni — APPROVED** [review]

**Pull request merged**